### DISCUSSION TITLE
/modes/train/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#8943] [Docs]
# /modes/train/

Step-by-step guide to train YOLOv8 models with Ultralytics YOLO including examples of single-GPU and multi-GPU training

https://docs.ultralytics.com//modes/train/

**Comment by UtkuEla** [comment]
I have started a training with 500 epochs and 50 early stopping. I occasionally get GPU CUDA is out of memory errors. So I have to resume training from the last saved points but it takes too much time. I tried to change the early stopping parameter like this:

results = model.train(resume= True, patience = 10, save= True) 

But it still says the early stopping is 50 in the printed training data.

How can I change the early stopping parameter after a training is started? If it is not possible, how can I finish the training and get the accuracy/validation graphs for the training?
